hackathon_site/dashboard/frontend/src/pages/Inventory/Inventory.tsx

```diff
@@ -117,17 +129,21 @@ const Inventory = () => {
                         <InventoryGrid />
                         <Divider className={styles.inventoryLoadDivider} />
                         <Typography variant="subtitle2" align="center" paragraph>
-                            SHOWING 100 OF 123 ITEMS
+                            {count > 0
+                                ? `SHOWING ${items.length} OF ${count} ITEMS`
+                                : "NO ITEMS FOUND"}
```

Comment:
The inventory grid component also shows "No items found". 

How about if there are no items, we remove the `Divider` above and remove the no items message from either the inventory or inventory grid component? The message in the inventory grid looks kinda wonky, so I'd suggest removing that one.

Will need tests for this in the inventory and inventory grid tests too.

Reply:
oh right, I forgot to test when there are no items!
What do we need to test in the inventory grid if we are removing that message?

Reply:
Also since we are removing the divider, in this case we can use getByTestId() right?

Reply:
For testing the inventory grid, you can probably just remove the test for the "no items found" message

For the divider, tbh I wouldn't bother unit testing it since it's a minor UI thing. But if you'd like to (I'm fine either way), then a test ID would be the way to go